x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/multi_page_layout/components/bottom_bar.tsx

```diff
@@ -139,21 +141,40 @@ export const CreatePackagePolicyFinalBottomBar: React.FC<{
             </EuiButtonEmpty>
           </EuiFlexItem>
         </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <EuiButton
-            color="success"
-            fill
-            size="m"
-            href={getHref('integration_details_assets', {
-              pkgkey,
-            })}
-          >
-            <FormattedMessage
-              id="xpack.fleet.confirmIncomingData.viewDataAssetsButtonText'"
-              defaultMessage="View assets"
-            />
-          </EuiButton>
-        </EuiFlexItem>
+        {!isK8s && (
+          <EuiFlexItem grow={false}>
+            <EuiButton
+              color="success"
+              fill
+              size="m"
+              href={getHref('integration_details_assets', {
+                pkgkey,
+              })}
+            >
+              <FormattedMessage
+                id="xpack.fleet.confirmIncomingData.viewDataAssetsButtonText'"
+                defaultMessage="View assets"
+              />
+            </EuiButton>
+          </EuiFlexItem>
+        )}
+        {isK8s && (
+          <EuiFlexItem grow={false}>
+            <EuiButton
+              color="success"
+              fill
+              size="m"
+              href={getAbsolutePath(
+                '/app/dashboards#/view/kubernetes-f4dc26db-1b53-4ea2-a78b-1bfab8ea267c'
```

Comment:
Is this a fixed path? any possibility that this id is going to change in the future?

Reply:
`kubernetes-f4dc26db-1b53-4ea2-a78b-1bfab8ea267c` is the id of the cluster overview dashboard as set in the https://github.com/elastic/integrations/blob/52dbf62bfa5c7197ed4fe2eb70779c1b03e080aa/packages/kubernetes/kibana/dashboard/kubernetes-f4dc26db-1b53-4ea2-a78b-1bfab8ea267c.json#L3792

I don't know if there is any other way to directly point to a specific dashboard. I think this will work and I am not expecting the dashboard id to change. 

I am bit concerned about the absolute path. Is there any possibility that this may change in case of the user specifying a different base path?

Reply:
On the there hand the code prepends it to the base path. 
```
return core.http.basePath.prepend(`${path}`);
```

Reply:
It is somethings like 
```
http://ip:5601/app/dashboards#/view/kubernetes-f4dc26db-1b53-4ea2-a78b-1bfab8ea267c